File: Java/Estrutura condicional/Lanchonete.java
import java.util.Locale;
import java.util.Scanner;

public class Lanchonete {
    public static void main(String[] args) {

        Locale.setDefault(Locale.US);
        Scanner sc = new Scanner(System.in);

        int codigo, qtde;
        double preco;

        System.out.print("Codigo do produto comprado: ");
        codigo = sc.nextInt();
        System.out.print("Quantidade comprada: ");
        qtde = sc.nextInt();

        preco = 0;

        switch (codigo) {
            case 1:
                preco = 5.00 * qtde;
                break;

            case 2:
                preco = 3.50 * qtde;
                break;

            case 3:
                preco = 4.80 * qtde;
                break;

            case 4:
                preco = 8.90 * qtde;
                break;

            case 5:
                preco = 7.32 * qtde;
                break;
        }

        System.out.println("Valor total a pagar: R$ " + String.format("%.2f", preco));

        sc.close();

    }
}

File: Python/Estrutura Condicional/Lanchonete.py
codigo: int; qtde: int
valor: float

codigo = int(input("Codigo do produto comprado: "))
qtde = int(input("Quantidade comprada: "))

if codigo == 1:
    valor = qtde * 5.00

elif codigo == 2:
    valor = qtde * 3.50

elif codigo == 3:
    valor = qtde * 4.80

elif codigo == 4:
    valor = qtde * 8.90

elif codigo == 5:
    valor = qtde * 7.32


print(f"Valor a pagar: R$ {valor:.2f}")